packages/mesos/buildinfo.json

```diff
@@ -3,9 +3,9 @@
   "docker": "dcos-builder",
   "single_source" : {
     "kind": "git",
-    "git": "https://github.com/mesosphere/mesos",
-    "ref": "76c87d55a78e8cbb380fc0c71388ccec53ed1dc8",
-    "ref_origin" : "dcos-1.7"
+    "git": "git@github.com:apache/mesos.git",
```

Comment:
Will Make this into a separate commit, and discard the commit once we are done testing here.